targets/targets.json

```diff
@@ -482,11 +482,11 @@
         "release_versions": ["2"],
         "device_name": "MK20DX256xxx7"
     },
-    "K22F": {
+    "MCU_K22F512": {
         "supported_form_factors": ["ARDUINO"],
```

Comment:
I'd say this belongs in the `K22F` target definition, since the shared MCU doesn't really have a form factor.